x/stake/keeper/delegation.go

```diff
@@ -449,6 +457,8 @@ func (k Keeper) getBeginInfo(ctx sdk.Context, params types.Params, valSrcAddr sd
 	minTime time.Time, height int64, completeNow bool) {
 
 	validator, found := k.GetValidator(ctx, valSrcAddr)
+	fmt.Println(validator.Status)
```

Comment:
please delete all the debugging code in this file (see https://github.com/cosmos/cosmos-sdk/pull/2405/commits/6ae62f689ef0c9400a040764ec244fa362975e5e)